addon/components/session/objective-list.hbs

```diff
@@ -0,0 +1,100 @@
+<div
+  class="session-objective-list"
+  data-test-session-objective-list
+  {{did-insert (perform this.load) @session @session.objectives}}
```

Comment:
same question as above. is the second argument passed in order to force a re-render?

Reply:
Yup. Exactly that. It's the thing that is being tracked, but the real state of it is a promise so I didn't want to await it here. Instead just let ember do it's thing which will mark this property as needing to be updated and call `load` again.